http-server/src/tests.rs

```diff
@@ -37,23 +39,22 @@ pub async fn server_with_context() -> SocketAddr {
 	let mut server = HttpServerBuilder::default().build("127.0.0.1:0".parse().unwrap()).unwrap();
 
 	let ctx = TestContext;
-	let mut rpc_ctx = RpcContextModule::new(ctx);
+	let mut rpc_module = RpcModule::new(ctx);
```

Comment:
maybe have to different `constructors` on RpcModule such as `with_context` and `default`

Reply:
What does `without_context` use as the context? `()`?

Reply:
yeah, maybe overkill